packages/payment/package.json

```diff
@@ -30,5 +30,8 @@
     "devDependencies": {
         "commit-and-tag-version": "^12.1.0",
         "typescript": "^5.3.2"
+    },
+    "dependencies": {
+        "@loopback/rest": "^13.1.5"
```

Comment:
Maybe this should be a peer dependency.  The request/response are being provided by the Evoke runtime and will be for whatever version Evoke is using, not the plugin's version.

Also, I believe all Loopback does is re-export these from `express`, so maybe get them from `express` instead of adding Loopback as a dependency.  And `express` request/response extend node's `http.IncomingMessage` and `http.ServerResponse`, so we could maybe even just use those and not need any dependency or worry about if e.g. we upgrade Loopback and it starts using a new major version of `express`.